tests/test_hs_restclient.py

```diff
@@ -313,6 +312,128 @@ def test_folder_delete(self):
 
         self.assertEqual(response['resource_id'], '511debf8858a4ea081f78d66870da76c')
         self.assertEqual(response['path'], 'model/initial')
-                
+
+
+class TestResourceCopy(unittest.TestCase):
+    @with_httmock(mocks.hydroshare.resourceCopy_post)
+    def test_resource_copy(self):
+        hs = HydroShare()
+        response = hs.resource('511debf8858a4ea081f78d66870da76c').copy()
```

Comment:
I'd like to move this to a pattern that's more consistent with the REST API itself, so we can more easily write clients in different languages, and so that users don't have to learn two different syntaxes to use hydroshare. That said, utility functions can be useful so if you want a copyResource() function I can make that as a passthrough to hs.resource(id).copy()